editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -620,6 +620,7 @@ export function SpecialSizeMeasurementsKeepDeepEquality(): KeepDeepEqualityCall<
         newSize.parentLayoutSystem,
         newSize.layoutSystemForChildren,
         newSize.providesBoundsForChildren,
+        newSize.display,
```

Comment:
I think display and htmlElementName are missing from the gigantic `areEqual` expression from line 594

Reply:
fixed